test/integration/templates/gem_layout_test.rb

```diff
@@ -0,0 +1,29 @@
+require "integration_test_helper"
+
+class GemLayoutTest < ActionDispatch::IntegrationTest
+  should "render the feedback form" do
+    Capybara.current_driver = Capybara.javascript_driver
+
+    visit "/templates/gem_layout.html.erb"
+
+    click_on "Report a problem with this page"
+    assert page.has_content?("Help us improve GOV.UK")
+    assert page.has_field?("What went wrong?")
+    # Regression test for scenario where wrong URL is set
+    url_input = page.find("form[action='/contact/govuk/problem_reports'] input[name=url]", visible: false)
+    assert_equal page.current_url, url_input.value
```

Comment:
This could be in its own `should` block, e.g. `should "allow user to report a problem with the page"`